packages/manager/apps/public-cloud/src/components/easter-eggs/index.js

```diff
@@ -0,0 +1,12 @@
+import angular from 'angular';
+
+import component from './easter-eggs.component';
+
+const moduleName = 'publicCloudEasterEggs';
+
+angular
+  .module(moduleName, [])
```

Comment:
You are also using `oui` and `angular-translate`

Reply:
Missing dependencies were added.